package.json

```diff
@@ -31,15 +31,12 @@
     "melon": "./bin/melon.js"
   },
   "scripts": {
-    "allchecks": "yarn clean && yarn build && yarn compile && yarn deploy && yarn lint && yarn test",
+    "allchecks": "yarn clean && yarn build && yarn compile && yarn lint && yarn test",
     "build": "tsc -p tsconfig.json && tscpaths -p tsconfig.json -s ./src -o ./lib > /dev/null",
     "bundle": "rollup -c",
-    "clean": "rimraf lib && rimraf out && yarn clean:logs",
-    "clean:logs": "rimraf logs/*.log",
```

Comment:
Yes, I changed the way logs are named. They are named by the process id now. Why? Because logging by timestamp generates multiple logs per test suite when they run in parallel.